assets/vsphere_cloud_config.yaml

```diff
@@ -4,9 +4,7 @@ metadata:
   namespace: openshift-cluster-csi-drivers
   name: vsphere-csi-config
 data:
-  cloud.conf: |
-    [Global]
-    cluster-id = "${CLUSTER_ID}"
-    [VirtualCenter "${VCENTER}"]
-    insecure-flag = "true"
-    datacenters = "${DATACENTERS}"
+  cloud.conf: ""
```

Comment:
Please add a comment that cloud.conf is loaded from csi_cloud_config.ini from the operator

Reply:
fixed